src/osip-core/src/test/java/edu/kit/pse/osip/core/io/files/ClientSettingsWrapperTest.java

```diff
@@ -245,4 +281,165 @@ public void testSaveSettings() throws IOException {
+        int interval = wrapperNullEntry.getFetchInterval(1122);
+        assertEquals(interval, 1122);
+    }
+    
+    /**
+     * Test getting invalid fetchInterval
+     */
+    @Test
+    public void testInvalidFetchInterval() {
+        int interval = wrapperInvalidEntry.getFetchInterval(1122);
+        assertEquals(interval, 1122);
+    }
+    
+    /**
+     * Test getting null overheating alarm
+     */
+    @Test
+    public void testNullOverheatingAlarm() {
+        boolean value = wrapperNullEntry.getOverheatingAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting invalid overheating alarm
+     */
+    @Test
+    public void testInvalidOverheatingAlarm() {
+        boolean value = wrapperInvalidEntry.getOverheatingAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting null undercooling alarm
+     */
+    @Test
+    public void testNullUnderCoolingAlarm() {
+        boolean value = wrapperNullEntry.getUndercoolingAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting invalid undercooling alarm
+     */
+    @Test
+    public void testInvalidUnderCoolingAlarm() {
+        boolean value = wrapperInvalidEntry.getUndercoolingAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting null overflow alarm
+     */
+    @Test
+    public void testNullOverflowAlarm() {
+        boolean value = wrapperNullEntry.getOverflowAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting invalid overflow alarm
+     */
+    @Test
+    public void testInvalidOverflowAlarm() {
+        boolean value = wrapperInvalidEntry.getOverflowAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting null underflow alarm
+     */
+    @Test
+    public void testNullUnderflowAlarm() {
+        boolean value = wrapperNullEntry.getUnderflowAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting invalid underflow alarm
+     */
+    @Test
+    public void testInvalidUnderflowAlarm() {
+        boolean value = wrapperInvalidEntry.getUnderflowAlarm(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting null temperature diagramm
+     */
+    @Test
+    public void testNullTempDiagramm() {
+        boolean value = wrapperNullEntry.getTempDiagram(TankSelector.valuesWithoutMix()[0], false);
+        assertFalse(value);
+    }
+    
+    /**
+     * Test getting invalid temperature diagramm
+     */
+    @Test
+    public void testInvalidTempDiagramm() {
+        boolean value = wrapperInvalidEntry.getTempDiagram(TankSelector.valuesWithoutMix()[0], false);
```

Comment:
Bitte alle `TankSelector.valuesWithoutMix()[0]` in dieser Datei mit `testTank` ersetzen